src/ssmDocument/commands/openDocumentItem.ts

```diff
@@ -49,18 +44,19 @@ export async function openDocumentItem(node: DocumentItemNode, awsContext: AwsCo
             localize(
                 'AWS.message.error.ssmDocumet.openDocument.could_not_open',
                 'Could not fetch and display document {0} contents',
+                'Please check logs for more details',
```

Comment:
Looks like this string would go inti {0} and the documentname would not be shown in the message

Reply:
Will move into message